spotify_client_credentials_flow.py

```diff
@@ -0,0 +1,28 @@
+""" Client Credentials Flow
+
+The Client Credentials flow is used in server-to-server authentication. Only endpoints that do not
+access user information can be accessed. The advantage here in comparison with requests to the Web
+API made without an access token, is that a higher rate limit is applied.
+https://spotipy.readthedocs.io/en/2.16.1/#client-credentials-flow"""
+
+from dotenv import load_dotenv
+
+load_dotenv()
+
+import spotipy
+from spotipy.oauth2 import SpotifyClientCredentials
+
+auth_manager = SpotifyClientCredentials()
+sp = spotipy.Spotify(auth_manager=auth_manager)
+
+playlists = sp.user_playlists("spotify")
+while playlists:
+    for i, playlist in enumerate(playlists["items"]):
+        print(
+            "%4d %s %s"
+            % (i + 1 + playlists["offset"], playlist["uri"], playlist["name"])
+        )
```

Comment:
I see that this code is ripped from https://spotipy.readthedocs.io/en/2.16.1/#api-reference. I see a couple of infelicities about it. This is the lesser of the two.

Using '%' for formatting is old-hat. These days we would likely write:
```python
print( f'''{i + 1 + playlists["offset"]:4} {playlist["uri"]} {playlist["name"]}''' )
```
And actually I would tend to write this, simply to document their meanings:
```python
offset = i + 1 + playlists["offset"]
uri = playlist["uri"]
name = playlist["name"]
print( f"{offset:4} {uri} {name}" )